assets/shapes.wgsl

```diff
@@ -565,7 +566,7 @@ fn sd_exponential_falloff(d: f32, size: f32, power: f32) -> f32 {
     return a;
 }
 
-fn sd_exponential_falloff_3(d: f32, size: f32) -> f32 {
+fn sd_exponential_falloff_3_(d: f32, size: f32) -> f32 {
```

Comment:
Why the trailing underscores?

Reply:
Shader functions can't end on number anymore due to https://github.com/bevyengine/naga_oil usage.